### PR TITLE
Adding pointer to Dataroc wiki page to README.md

### PR DESCRIPTION
**Pull request by davidadamsphd**

### CODE REVIEW

**Comment by davidadamsphd** [comment]
This addresses #975

File: README.md [hunk 1]
```diff
@@ -41,10 +41,21 @@ To run a single test class, run something like this, `gradle test -Dtest.single=
...
 
 Note: for faster gradle operations, add `org.gradle.daemon=true` to your `~/.gradle/gradle.properties` file.  This will keep a gradle daemon running in the background and avoid the ~6s gradle start up time on every command.  
 
+
+Running GATK 4
+------------
+To run the main program locally, run `build/install/gatk/bin/gatk`.
+
+For spark tools, first build the Spark jar using `gradle sparkJar`. There are three main ways to run gatk 4 with Spark,
+* Locally, using the regular gatk jar and `--master=local[2]`
```
Comment (lbergelson) [inline README.md]: Should probably specify `local[*]` instead of `local[2]` use the number of cores you have available.

File: README.md [hunk 2]
```diff
@@ -41,10 +41,21 @@ To run a single test class, run something like this, `gradle test -Dtest.single=
...
-To run the main program, run `build/install/gatk/bin/gatk`.
 
 Note: for faster gradle operations, add `org.gradle.daemon=true` to your `~/.gradle/gradle.properties` file.  This will keep a gradle daemon running in the background and avoid the ~6s gradle start up time on every command.  
 
+
+Running GATK 4
+------------
+To run the main program locally, run `build/install/gatk/bin/gatk`.
+
+For spark tools, first build the Spark jar using `gradle sparkJar`. There are three main ways to run gatk 4 with Spark,
```
Comment (lbergelson) [inline README.md]: mention where the generated sparkJar will be (in build/libs/) and it's name (something-SNAPSHOT-spark.jar)

**Comment by lbergelson** [comment]
two minor nitpicks, feel free to squash and merge when those are updated

**Comment by lbergelson** [comment]
@droazen and I haven't had any luck getting the proxy working following your instructions.  We'll need to bother you about it in person and figure out whats missing.

**Pull request merged**